package.json

```diff
@@ -34,14 +34,14 @@
     "api-report": "node scripts/api-report.js",
     "predocs": "npm run compile",
     "docs": "jsdoc -c .jsdoc.js",
-    "system-test:rest": "FIRESTORE_PREFER_REST=true mocha build/system-test --timeout 600000",
-    "system-test:named-db:rest": "FIRESTORE_NAMED_DATABASE=test-db FIRESTORE_PREFER_REST=true mocha build/system-test --timeout 600000",
-    "system-test:grpc": "mocha build/system-test --timeout 600000",
-    "system-test:named-db:grpc": "FIRESTORE_NAMED_DATABASE=test-db mocha build/system-test --timeout 600000",
-    "system-test:emulator:rest": "FIRESTORE_EMULATOR_HOST=localhost:8080 FIRESTORE_PREFER_REST=true mocha build/system-test --timeout 600000",
-    "system-test:named-db:emulator:rest": "FIRESTORE_NAMED_DATABASE=test-db FIRESTORE_EMULATOR_HOST=localhost:8080 FIRESTORE_PREFER_REST=true mocha build/system-test --timeout 600000",
-    "system-test:emulator:grpc": "FIRESTORE_EMULATOR_HOST=localhost:8080 mocha build/system-test --timeout 600000",
-    "system-test:named-db:emulator:grpc": "FIRESTORE_NAMED_DATABASE=test-db FIRESTORE_EMULATOR_HOST=localhost:8080 mocha build/system-test --timeout 600000",
+    "system-test:rest": "FIRESTORE_PREFER_REST=true mocha build/system-test --timeout 1200000",
```

Comment:
why the timeout is extended? would this slow the full cycle down?

Reply:
yeah it'll slow down the start-to-finish time of the "integration tests CI". This PR adds 120 tests and the tests take a while because each test does at the very least one round trip to Firestore server, and at least 2-3 round-trips to cloud trace server (because the traces don't become available in the cloud trace server immediately, so we need to retry).

I am open to suggestions about how to reduce this cost. Maybe we should run the in-memory tests on PR CI and cloud-trace backend tests on a nightly basis.

Reply:
if only 10 minutes are added on top to run the whole system test, I think we could live with that comparing to having some part of the code not tested in PRs.